sys/drv_stdvga.c

```diff
@@ -145,23 +144,35 @@ static int stdvga_fb_write(vga_device_t *vga, uio_t *uio) {
   return 0;
 }
 
-int stdvga_pci_attach(pci_device_t *pci) {
-  if (pci->vendor_id != VGA_QEMU_STDVGA_VENDOR_ID ||
-      pci->device_id != VGA_QEMU_STDVGA_DEVICE_ID)
+static int stdvga_probe(device_t *dev) {
+  pci_dev_data_t *pcid = dev->instance;
+
+  if (pcid == NULL)
+    return 0;
```

Comment:
What is the purpose of this test? I suppose it might be turned into `assert(pcid != NULL)`, `instance` will always be set (to something) when we're probing a device.